[CUBRIDMAN-235] CUBRID_TMP description (#537)
CUBRID/cubrid-manual: en/env.rst

    *   A user of CUBRID Manager should specify **CUBRID_MSG_LANG**, an environment variable of DB server node into **en_US** to print out messages normally after running database related features. However, database related features are run normally and just the output messages are broken when **CUBRID_MSG_LANG** is not **en_US**.
    *   To apply the changed **CUBRID_MSG_LANG**, CUBRID system of DB server node should be restarted(cubrid service stop, cubrid service start).

*   **CUBRID_TMP**: The environment variable that specifies the location where CUBRID stores temporary files. There are two types of temporary files: regular files and UNIX domain sockets. If specified, temporary files are created in the directory specified in the environment variable CUBRID_TMP.

    *   **Regular files**: Temporary files are created by utilities such as **plandump** in addition to the database server and csql. If this environment variable is not set, the following directory is used.

        * Linux: /tmp
        * Windows: C:\\Windows\\TEMP

>     *   **UNIX Domain Socket files**: if this environment variable is not set, UNIX Domain sockets will be created in the directory for each process as follow (Applicable to **Linux** only and does not apply to **Windows**):

Reviewer: ```suggestion
    *   **UNIX Domain Socket files**: If this environment variable is not set, UNIX Domain sockets will be created in the following directories for each process (Applicable to **Linux** only and does not apply to **Windows**):
```

    
        *   cub_master: **/tmp**
        *   cub_broker process: **$CUBRID/var/CUBRID_SOCK**
        *   cub_javasp process: **$CUBRID/var/CUBRID_SOCK**

.. note::

>     * If the **CUBRID_TMP** environment variable is set, value of **java.io.tmpdir** among **java_stored_procedure_jvm_options** is ignored. Which is an option used by the cub_javasp process (built-in Java VM) to store temporary files, instead it will use the directory specified by CUBRID_TMP.

Reviewer 2: ```suggestion
    * **java.io.tmpdir** among **java_stored_procedure_jvm_options** is an option that specifies the directory where temporary files of the cub_javasp process (built-in Java VM) are created. If the **CUBRID_TMP** environment variable is set, value of **java.io.tmpdir** is ignored, instead it will use the directory specified by CUBRID_TMP. 
```

Author: comment 감사합니다.

>     * **java.io.tmpdir** among **java_stored_procedure_jvm_options** is an option that specifies the directory where temporary files of the cub_javasp process (built-in Java VM) are created. If the **CUBRID_TMP** environment variable is set, value of **java.io.tmpdir** is ignored, instead it will use the directory specified by CUBRID_TMP. 

Reviewer: ```suggestion
    * **java.io.tmpdir** among **java_stored_procedure_jvm_options** is an option that specifies the directory where temporary files of the cub_javasp process (built-in Java VM) are created. If the **CUBRID_TMP** environment variable is set, the value of **java.io.tmpdir** is ignored, and the directory specified by CUBRID_TMP will be used instead.
```

Author: Thank you @junsklee , much better.
